### DISCUSSION TITLE
Smart Check for UI Properties within GeoNodes

### DISCUSSION THREAD

**Discussion by Realium3D** [Mustard2/MustardUI#98] [Ideas]
I'm currently using a GeoNode modifier for custom eyelashes on my character model and have a number of settings that I have to manually add to the UI. I'm not sure how feasible it would be or if there are any performance issues but it would be useful if the Properties Smart Checker could find values using the "MustardUI Float - " inside a Geometry Node.

As the lashes mesh/s are separate from the main body I'm not sure if the smart checker would need to be told which mesh/s the GeoNodes are active on first.

**Comment by Mustard2** [comment]
Might be an idea, yes!

Or I can try to add a full support for geonodes, with the possibility to show the settings of those in the settings. 

Sorry if I didn't answer before, I don't receive notifications for discussions. :(
You can also add issues, those receive notifications!

**Comment by Mustard2** [comment]
Solved in #119
